### PR TITLE
Getting-started guide: Linked to AWS docs and did a tech writer review

### PR DESCRIPTION
**Pull request by sarahmaddox**
fixes https://github.com/kubeflow/website/issues/670
fixes https://github.com/kubeflow/website/issues/671

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/672)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
This PR is ready for review. Adding a hold to allow for multiple reviewers.

/hold

**Comment by sarahmaddox** [comment]
Preview: https://deploy-preview-672--competent-brattain-de2d6d.netlify.com/docs/started/getting-started/

File: content/docs/started/getting-started.md [hunk 1]
```diff
@@ -1,63 +1,72 @@
 +++
 title = "Getting Started with Kubeflow"
-description = "Quickly get running with your ML Workflow"
+description = "Get your machine-learning workflow up and running on Kubeflow"
 weight = 1
 +++
 
-There are a variety of ways to install Kubeflow
+There are various ways to install Kubeflow. Choose one of the following
```
Comment (dwightjl) [inline content/docs/started/getting-started.md]:
```suggestion
There are various ways to install Kubeflow. Choose one of the following options
```

File: content/docs/started/getting-started.md [hunk 2]
```diff
@@ -1,63 +1,72 @@
...
+to suit your environment (cloud, on premises (on prem) or local):
```
Comment (dwightjl) [inline content/docs/started/getting-started.md]:
```suggestion
to suit your environment (cloud, on premises (on prem), or local):
```

File: content/docs/started/getting-started.md [hunk 3]
```diff
@@ -1,63 +1,72 @@
...
-	     (e.g. [Hypervisor
-	     Framework](https://developer.apple.com/documentation/hypervisor) on
-	     MacOS, [Hyper-V on Windows
-	     10](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v),
-	     QEMU/KVM for linux)
-	   - Eliminates the need to install a separate virtualization
-	     application.
-	   - You can use cloud-init to customize the VM (as you might in a cloud)
+   * [MicroK8s setup](/docs/started/getting-started-multipass/)
+      * The benefits of using [MicroK8s](https://microk8s.io/) include the
```
Comment (dwightjl) [inline content/docs/started/getting-started.md]:
```suggestion
      * [MicroK8s](https://microk8s.io/) can provide the following benefits:
```

File: content/docs/started/getting-started.md [hunk 4]
```diff
@@ -1,63 +1,72 @@
...
+        following:
```
Comment (dwightjl) [inline content/docs/started/getting-started.md]:
```suggestion

```

File: content/docs/started/getting-started.md [hunk 5]
```diff
@@ -1,63 +1,72 @@
...
+          - A small, fast, secure, single node Kubernetes that installs on any
```
Comment (dwightjl) [inline content/docs/started/getting-started.md]: Perhaps say "single node Kubernetes pod" or "single node Kubernetes configuration"?

File: content/docs/started/getting-started.md [hunk 6]
```diff
@@ -1,63 +1,72 @@
...
+            Linux system as a [snap](https://snapcraft.io/microk8s).
+          - Strong isolation and update semantics - your cluster
+            is updated within a short period after upstream Kubernetes
+            releases.
+          - Built-in support for enabling an already-installed GPU:
```
Comment (dwightjl) [inline content/docs/started/getting-started.md]:
```suggestion
          - Built-in support to enable an installed GPU:
```

Comment (dwightjl) [reply content/docs/started/getting-started.md]: Minor changes to simplify the statement.

**Comment by Jeffwan** [comment]
Thanks a lot for helping add aws section in getting-started doc. @sarahmaddox

File: content/docs/started/getting-started.md (hunk 5 above)
Comment (sarahmaddox) [reply content/docs/started/getting-started.md]: Done

**Pull request merged**